src/Asset/Controller/Grid/Configuration/GetAvailableColumnsController.php

```diff
@@ -51,20 +51,20 @@ public function __construct(
      * @throws NotFoundException|SearchException
      */
     #[Route(
-        '/assets/grid/available-configuration',
+        '/assets/grid/available-columns',
         name: 'pimcore_studio_api_get_asset_grid_available_configuration',
         methods: ['GET']
     )]
     #[IsGranted(UserPermissions::ASSETS->value)]
     #[Get(
-        path: self::API_PATH . '/assets/grid/available-configuration',
+        path: self::API_PATH . '/assets/grid/available-columns',
         operationId: 'asset_get_available_grid_configuration',
```

Comment:
```suggestion
        operationId: 'asset_get_available_grid_columns',
```

Reply:
Already Fixed it here :D
https://github.com/pimcore/studio-backend-bundle/pull/390/commits/21546aea5af75d7cbdc5d9e8fc7b8613e4ed4a2b
https://github.com/pimcore/studio-backend-bundle/pull/390/commits/e824b828233afe7b44caf644aba9685efe9fee16